sips/sip-026/sip-026-sbtc_peg.md

```diff
@@ -0,0 +1,166 @@
+# sBTC SIP
+
+## SIP Number: 022
```

Comment:
```suggestion
## SIP Number: 028
```

Reply:
Additionally, the path to the files should be modifed from `sip-026`->`sip-028` per @Hero-Gamer